reference/7.1/Microsoft.PowerShell.Utility/Sort-Object.md

```diff
@@ -579,17 +581,23 @@ You can pipe the objects to be sorted to `Sort-Object`.
 ## NOTES
 
 The `Sort-Object` cmdlet sorts objects based on properties specified in the command or the default
-sort properties for the object type. If an object does not have one of the specified properties, the
-property value for that object is interpreted by `Sort-Object` as **Null** and placed at the end of
-the sort order.
+sort properties for the object type. Default sort properties are defined using the
+`DefaultKeyPropertySet` element in a `types.ps1xml` file. For more information, see
```

Comment:
using the `PropertySet` named `DefaultKeyPropertySet` in a

Reply:
Fixed it.